### DISCUSSION TITLE
Bookmark applications.

### DISCUSSION THREAD

**Discussion by the-harry** [argoproj/argo-cd#7361] [Ideas]
Hello guys, I've started to use Argo recently in my job, and it's amazing! But one thing that I missed was the ability to bookmark my most used applications so they always show up first when I sign in.
I always use the search bar, which is fine, but would be really nice if we could create bookmarks for each user.

Is there anything like this in the roadmap? What do you guys think? Is it feasible?

**Comment by alexec** [comment]
ArgoCD?

**Reply by the-harry** [reply]
Yes, exactly @alexec, ArgoCD.
